packages/shared/src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -40,16 +40,18 @@ export const RichTextEditor = ({
 
     if (isEditing) {
         return (
-            <FondueRichTextEditor
-                id={id}
-                value={value}
-                border={false}
-                placeholder={placeholder}
-                plugins={plugins}
-                updateValueOnChange={updateValueOnChange}
-                onValueChanged={() => setShouldPreventPageLeave(true)}
-                onTextChange={saveText}
-            />
+            <div className="[&_[contenteditable]]:tw-outline-none">
```

Comment:
Just wondering why this div is needed?

Reply:
New RTE adds a blue outline which was not there previously.